### PR TITLE
Add option to scan certificates on nodes

### PR DESCRIPTION
**Pull request by DebakelOrakel**
## Checklist

- [x] PR contains a single logical change (to build a better changelog).
- [x] Update the documentation.
- [x] Categorize the PR by setting a good title and adding one of the labels:
      `bug`, `enhancement`, `documentation`, `change`, `breaking`, `dependency`
      as they show up in the changelog.
- [x] Link this PR to related issues or PRs.

Closes #1

### CODE REVIEW

File: tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml [hunk 1]
```diff
@@ -25,3 +25,15 @@ rules:
...
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/instance: cert-exporter
+    app.kubernetes.io/managed-by: Helm
+    app.kubernetes.io/name: x509-certificate-exporter
+    app.kubernetes.io/version: 2.12.0
+    helm.sh/chart: x509-certificate-exporter-1.20.0
+  name: cert-exporter-x509-certificate-exporter-node
+rules: null
```
Comment (megian) [inline tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml:39]: If the ClusterRole has no rules, is she required?

File: class/defaults.yml [hunk 2]
```diff
@@ -13,12 +13,18 @@ parameters:
 
     ignore_alerts: []
 
+    # secrets exporter configs
     include_namespaces: []
-    exclude_namespaces:
-      - openshift-operator-lifecycle-manager
```
Comment (megian) [inline class/defaults.yml:18]: What is the reason for this?

**Review by megian — CHANGES_REQUESTED** [review]

File: tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml (hunk 1 above)
Comment (DebakelOrakel) [reply tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml:39]: This is a generated yaml from the tests.
But good question; this is generated from helm 🤷

File: class/defaults.yml (hunk 2 above)
Comment (DebakelOrakel) [reply class/defaults.yml:18]: I figured we should manage these params in the commodore defaults.
Mainly bc it should be the same over all distributions and we don't wan't to manage this per cluster.
We should however have a discussion on the commodore defaults an what should go in there.

File: docs/modules/ROOT/pages/references/parameters.adoc [hunk 3]
```diff
@@ -64,6 +64,54 @@ default::
 Exclude TLS Secrets having those labels.
 Items can be keys such as `my-label` or also require a value with the syntax `my-label=my-value`.
 
+== `watch_dirs`
+
+[horizontal]
+type:: list
+default::
++
+[source,yaml]
```
[the remark this answers is not in the document]
Comment (DebakelOrakel) [reply docs/modules/ROOT/pages/references/parameters.adoc:73]: Ah yes, will change that.

File: class/defaults.yml (hunk 2 above)
Comment (megian) [reply class/defaults.yml:18]: Sounds reasonable.

File: tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml (hunk 1 above)
Comment (megian) [reply tests/golden/defaults/cert-exporter/cert-exporter/10_helmchart/x509-certificate-exporter/templates/clusterrole.yaml:39]: If it's generated, it makes sense to keep it as is and not manually changing it. Fine by me.

**Pull request merged**